src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/HttpGoogleBillingDAO.scala

```diff
@@ -383,6 +394,18 @@ class HttpGoogleBillingDAO(appName: String,
     }
   }
 
+  def cleanupClusters(projectName: String): Future[Unit] = {
+    for {
+      result <- googleRq(dataproc.projects().regions().clusters().list(projectName, "us-central1"))
```

Comment:
Maybe put a comment in code mentioning that Leo currently only creates clusters in us-central1.

Reply:
I'd also advise putting a comment in Leo code telling the reader to update gpalloc if this changes :)